src/Sylius/Bundle/AdminBundle/Resources/config/routing.yml

```diff
@@ -79,7 +79,7 @@ sylius_admin_product_review:
 
 sylius_admin_product_taxon:
     resource: "@SyliusAdminBundle/Resources/config/routing/product_taxon.yml"
-    prefix: /product-taxons
+    prefix: /taxons/{taxonId}/products/
```

Comment:
🤔 
```suggestion
    prefix: /taxons/{id}/products
```

Reply:
It's a product taxon subresource entry point, using id would be confusing if you add any kind of operation inside, you'd have id = taxon, smthelseId = productTaxon;
We have it set up the same way for variants and coupons.